exetera/core/dataframe.py

```diff
@@ -435,6 +435,40 @@ def apply_index(self, index_to_apply, ddf=None):
                 field.apply_index(index_to_apply, in_place=True)
             return self
 
+    def sort_values(self, by, ascending=True, ddf=None):
```

Comment:
Can we put the type info on these parameters please?

Reply:
ascending isn't currently used anywhere. I understand that this is due to dataset_sort_index not accepting one